app/src/main/java/cm/aptoide/pt/home/HomeContainerNavigator.java

```diff
@@ -9,37 +9,59 @@
 
 public class HomeContainerNavigator {
 
-  private FragmentNavigator fragmentNavigator;
+  private FragmentNavigator childFragmentNavigator;
+  private String homeTag;
+  private String gamesTag;
+  private String appsTag;
 
-  public HomeContainerNavigator(FragmentNavigator fragmentNavigator) {
-    this.fragmentNavigator = fragmentNavigator;
+  public HomeContainerNavigator(FragmentNavigator childFragmentNavigator) {
+    this.childFragmentNavigator = childFragmentNavigator;
   }
 
   public void loadMainHomeContent() {
-    fragmentNavigator.navigateToWithoutBackSave(new HomeFragment(), true);
+    Fragment fragment = childFragmentNavigator.getFragment(homeTag);
+    if (fragment != null) {
+      childFragmentNavigator.navigateToWithoutBackSave(fragment, true);
+    } else {
+      homeTag = childFragmentNavigator.navigateTo(new HomeFragment(), true);
+    }
```

Comment:
I don't think it's really needed but eventually it might make sense to extract this navigation behaviour to FragmentNavigator, especially if we have a similar behaviour with BN in the future. But for now I think it's fine